internal/deviations/deviations.go

```diff
@@ -486,6 +486,25 @@ func StorageComponentUnsupported(_ *ondatra.DUTDevice) bool {
 	return *storageComponentUnsupported
 }
 
+// PlatformPowerEnableWait returns timer value to wait while platform is powered on.
+// Default value is 0.
+func PlatformPowerEnableWait(dut *ondatra.DUTDevice) int64 {
+	logErrorIfFlagSet("deviation_platform_power_enable_wait")
+	ppewait := lookupDUTDeviations(dut).GetPlatformPowerEnableWait()
+	var min_ppwewait int64 = 5
+	if min_ppwewait < ppewait {
+		return min_ppwewait
+	}
+	return ppewait
+}
+
+// PlatformPowerDownUpSkip returns whether the device should skip the Platform Power Down Up case.
+// Default value is false.
+func PlatformPowerDownUpSkip(dut *ondatra.DUTDevice) bool {
+	logErrorIfFlagSet("deviation_platform_power_down_up_skip")
```

Comment:
the `logErrorIfFlagSet` line can be removed from both functions. Since this is a new deviation the flag value is not avialable and we do not expect anyone setting these deviation via flag